apps/web/components/Celebration.tsx

```diff
@@ -4,17 +4,44 @@ import Image from "next/image";
 import { getCelebrationImage } from "@/utils/celebration";
 import { Button } from "@/components/Button";
 
-export function Celebration(props: { message: string }) {
+export function Celebration(props: { type: string }) {
   const [active, setActive] = useState(false);
 
   useEffect(() => {
     setActive(true);
   }, []);
 
+  if (props.type == "draft") {
+    return (
+      <>
+        <div className="mt-20 flex items-center justify-center font-cal text-2xl text-gray-900">
```

Comment:
This style is copy passed each time. But either way, just pass in the message and don't edit this component.